public-api/v1alpha/lib/pipelines_api/periodic_scheduler_client/response_formatter.ex

```diff
@@ -152,6 +156,27 @@ defmodule PipelinesAPI.PeriodicSchedulerClient.ResponseFormatter do
     DateTime.to_string(ts_date_time)
   end
 
+  defp rename_reference_to_branch(periodics) when is_list(periodics) do
+    Enum.map(periodics, &rename_reference_to_branch/1)
+  end
+
+  defp rename_reference_to_branch(periodic) do
+    case reference_to_branch_field(periodic.reference) do
+      nil -> periodic
+      branch_name -> Map.put(periodic, :branch, branch_name)
+    end
+  end
+
+  def reference_to_branch_field(reference) when is_binary(reference) do
+    with true <- String.starts_with?(reference, "refs/heads/"),
+         true <- not String.starts_with?(reference, "refs/pull/"),
+         true <- not String.starts_with?(reference, "refs/tags/") do
```

Comment:
The last two `String.starts_with?` checks are redundant, since if the first one is true, the rest will always be false.